src/api/spec/factories/packages.rb

```diff
@@ -28,5 +28,14 @@
         Suse::Backend.put("/source/#{URI.escape(package.project.name)}/#{URI.escape(package.name)}/_service", '<service/>')
       end
     end
+
+    factory :package_with_failed_comment_attribute do
+      after(:create) do |package|
+        attribute_type = AttribType.find_by_name("OBS:ProjectStatusPackageFailComment")
+        attrib = build(:attrib, attrib_type: attribute_type, package: package)
```

Comment:
You can create :attrib factories without any value... See those:

https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/models/user_spec.rb#L124

https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/controllers/webui/search_controller_spec.rb#L12

https://github.com/openSUSE/open-build-service/blob/master/src/api/spec/factories/project.rb#L57